### DISCUSSION TITLE
Lineage for glue dynamicframe

### DISCUSSION THREAD

**Discussion by rushabh1995** [AbsaOSS/spline#1303] [Q&A]
Hi guys,
I am trying to extract lineage from Glue job using spline JAR **(spark-3.3-spline-agent-bundle_2.12 JAR 2.0.0)** which works only for spark DataFrame but not glue dynamic frame.
Is there any functionality provided in spline JAR or anything which will be **helpful** to extract the lineage of Glue DynamicFrame.
Currently, we are using Glue 4.0 version for our UseCase.

**Comment by wajda** [comment]
closing as a duplicate - #1304
